client/blocks/plugins-update-manager/schedule-form.helper.ts

```diff
@@ -0,0 +1,104 @@
+export const MAX_SELECTABLE_PLUGINS = 10;
+
+export const DAILY_OPTION = {
+	label: 'Daily',
+	value: 'daily',
+};
+
+export const WEEKLY_OPTION = {
+	label: 'Weekly',
+	value: 'weekly',
+};
+
+export const DAY_OPTIONS = [
+	{
+		label: 'Monday',
+		value: 'a',
+	},
+	{
+		label: 'Tuesday',
+		value: 'b',
+	},
+	{
+		label: 'Wednesday',
+		value: 'c',
+	},
+	{
+		label: 'Thursday',
+		value: 'd',
+	},
+	{
+		label: 'Friday',
+		value: 'e',
+	},
+	{
+		label: 'Saturday',
+		value: 'f',
+	},
+	{
+		label: 'Sunday',
+		value: 'g',
+	},
+];
+
+export const HOUR_OPTIONS = [
+	{
+		label: '00',
+		value: '0',
+	},
+	{
+		label: '01',
+		value: '1',
+	},
+	{
+		label: '02',
+		value: '3',
+	},
+	{
+		label: '03',
+		value: '3',
+	},
+	{
+		label: '04',
+		value: '4',
+	},
+	{
+		label: '05',
+		value: '5',
+	},
+	{
+		label: '06',
+		value: '6',
+	},
+	{
+		label: '07',
+		value: '7',
+	},
+	{
+		label: '08',
+		value: '8',
+	},
+	{
+		label: '09',
+		value: '9',
+	},
+	{
+		label: '10',
+		value: '10',
+	},
+	{
+		label: '11',
+		value: '11',
+	},
```

Comment:
How will we display 12 am or 12 pm? Is there not a component that we can re-use for this?

Reply:
That list is only a placeholder, and you are right, I missed the 12 option. I just working on integrating the CRUD actions, so It will be done in the next PR: https://github.com/Automattic/wp-calypso/pull/87897